docs/tutorials/excel-tutorial.md

```diff
@@ -22,6 +22,8 @@ In this tutorial, you'll create an Excel task pane add-in that:
 
 > [!TIP]
 > If you've already completed the [Build an Excel task pane add-in](../quickstarts/excel-quickstart-jquery.md) quick start using the Yeoman generator, and want to use that project as a starting point for this tutorial, go directly to the [Create a table](#create-a-table) section to start this tutorial.
+>
+> If you want a completed version of this tutorial, head over to the [Office Add-ins samples repo on GitHub](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/tutorials/excel-tutorial).
 
 ## Prerequisites
 
```

Comment:
Consider including the instruction to run "npm install", perhaps at the end of the "Create your add-in project" section. Without running that instruction, I ended hitting the following error when I tried to test the add-in later on:
```
'office-addin-debugging' is not recognized as an internal or external command,
operable program or batch file.
```